src/tests/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from '../App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] You currently have only one test whereas you should have more tests. Kindly add more tests for your app components.

  ![reem-lab-tests-2](https://user-images.githubusercontent.com/36542986/156489397-e010c8c6-4de4-48d9-9dca-0c85fb8f5f80.jpg)

- [ ] Your test is failing. Kindly fix it.

  ![reem-lab-tests](https://user-images.githubusercontent.com/36542986/156489264-278c696e-7d71-4f0a-a939-97c8b4782577.jpg)